### PR TITLE
ICU-22325 integrate CLDR release 44 alpha2 to ICU

### PR DESCRIPTION
**Pull request by DraganBesevic**
##### Checklist

- [ ] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-22325
- [ ] Required: The PR title must be prefixed with a JIRA Issue number. 
- [ ] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [ ] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [ ] Issue accepted (done by Technical Committee after discussion)
- [ ] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

Integrated CLDR release-44-alpha2 

This is the integration with data from the CLDR 44 after data freeze

This PR contains multiple commits:

1. Binaries, binary-as-source. (no review needed)
2. Data sources from CLDR. (can spot check)
3. ICU lib/tool source, plus test data & dtd copied from CLDR.

New logKnownIssue test skips:

- [CLDR-17028](https://unicode-org.atlassian.net/browse/CLDR-17028) ExhaustivePersonNameFormatterTest unit tests fails for many locales (additional locales)

ALLOW_MULTIPLE_COMMITS=true

**Note: When it is time to merge this, squash together just the part 3 commits (if more than one), but keep the part 1 and part 2 commits separate. Then use the normal rebase and merge button to merge as 3 separate commits.**

[CLDR-17028]: https://unicode-org.atlassian.net/browse/CLDR-17028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by macchiati — DISMISSED** [review]
These a far easier to review by being split into functional commits!!

File: icu4c/source/data/brkitr/root.txt [hunk 1]
```diff
@@ -22,8 +22,8 @@ root{
         Mymr:process(dependency){"burmesedict.dict"}
         Thai:process(dependency){"thaidict.dict"}
     }
-    lstm{
+   lstm{
         Thai{"Thai_graphclust_model4_heavy.res"}
         Mymr{"Burmese_graphclust_model5_heavy.res"}
-    }
+    }
```
Comment (markusicu) [inline icu4c/source/data/brkitr/root.txt:28]: Funky whitespace reformatting. Is this from the CLDR-to-ICU converter?

File: icu4c/source/data/curr/ar.txt [hunk 2]
```diff
@@ -763,9 +763,13 @@ ar{
...
             "كرونة سلوفاكية",
         }
+        SLE{
+            "SLE",
+            "ليون سيراليوني",
+        }
         SLL{
             "SLL",
-            "ليون سيراليوني",
+            "ليون سيراليوني - 1964-2022",
```
Comment (markusicu) [inline icu4c/source/data/curr/ar.txt:772]: Just FYI:
I just realized that we occasionally have hardcoded numbers like this in display names. This is a bit problematic in locales where we can expect users to select different numbering systems (in this case Latn vs. Arab).

**Review by markusicu — APPROVED** [review]

File: icu4c/source/data/brkitr/root.txt (hunk 1 above)
Comment (DraganBesevic) [reply icu4c/source/data/brkitr/root.txt:28]: It could just be my editor!

Comment (markusicu) [reply icu4c/source/data/brkitr/root.txt:28]: Is it a manual edit? If it's manual, I would expect it to be aligned nicely...

Comment (pedberg-icu) [reply icu4c/source/data/brkitr/root.txt:28]: It is copy and paste of text from section 5e of the integration instructions: https://unicode-org.github.io/icu/processes/cldr-icu.html#5-generate-cldr-production-data-and-convert-for-icu

File: icu4c/source/data/curr/ar.txt (hunk 2 above)
Comment (pedberg-icu) [reply icu4c/source/data/curr/ar.txt:772]: Sounds like a godd topic for a CLDR ticket... thanks!

**Comment by pedberg-icu** [comment]
@markusicu @macchiati The two part-3 commits were squashed, please re-approve, thanks! And the previously-passing flaky maven tests are now failing again, will re-run until they pass. (Exhaustive also previously passed)

**Review by markusicu — APPROVED** [review]

**Pull request merged**